src/fa/replay.py

```diff
@@ -87,7 +87,15 @@ def replay(source, detach=False):
             if url.scheme() == "faflive":
                 mod = QtCore.QUrlQuery(url).queryItemValue("mod")
                 mapname = QtCore.QUrlQuery(url).queryItemValue("map")
-                replay_id = url.path().split("/")[0]
+                replay_id = url.path().split("/")[1]
+                if int(replay_id) not in client.instance.gameset:
+                    if QtWidgets.QMessageBox.question(client.instance, "Live Game ended",
+                                                      "Would you like to look for it in Replays?",
+                                                      QtWidgets.QMessageBox.Yes,
+                                                      QtWidgets.QMessageBox.No) == QtWidgets.QMessageBox.Yes:
+                        return "Replays"
```

Comment:
Return type doesn't match up.

Reply:
aren't we flexible there?

Reply:
This is not an excuse. This return type means something to the caller, adding something out of the blue is unacceptable.